src/pages/settings/InitialSettingsPage.js

```diff
@@ -168,13 +173,55 @@ class InitialSettingsPage extends React.Component {
                 iconFill: themeColors.iconReversed,
                 fallbackIcon: Expensicons.FallbackWorkspaceAvatar,
                 brickRoadIndicator: PolicyUtils.getPolicyBrickRoadIndicatorStatus(policy, this.props.policyMembers),
+                pendingAction: policy.pendingAction ? policy.pendingAction : null,
```

Comment:
The ternary here is useless, no?

Reply:
Yes good point. I was thinking it would throw an error if undefined, but looks like that is not the case.